jwst/mrs_imatch/mrs_imatch_step.py

```diff
@@ -29,14 +20,27 @@ class MRSIMatchStep(Step):
         bkg_degree = integer(min=0, default=1) # Degree of the polynomial for background fitting
         subtract = boolean(default=False) # subtract computed sky from 'images' cube data?
         skip = boolean(default=True) # Step must be turned on by parameter reference or user
-    """ # noqa: E501
+    """  # noqa: E501
 
     reference_file_types: list = []
 
     def process(self, images):
+        """
+        Run this step for a given list of images.
 
+        Parameters
+        ----------
+        images : file path or list of DataModels
+            If a file path, initialize from an association table.
+            If a list, can be a list of DataModels of any type
```

Comment:
Is it really true the datamodels can be of any type?  I see in the docstring of `apply_background_2d` that these need to be ImageModels, and I don't see anywhere that non-imagemodels get removed from the ModelContainer

Reply:
I agree no point diving too much, but `list[obj]` is even less specific than what's there currently.  Certainly it needs to at least have datamodels as input to do anything meaningful

Reply:
OK I change "DataModels" to "ImageModels".